### PR TITLE
Add node name as a placeholder

### PR DESCRIPTION
**Pull request by pxaws**
**Description:** <Describe what has changed.>
To support aws container insight use case, we add `{NodeName}` as a placeholder so that the emf logs from each node will be published to its corresponding log stream. 

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2307

**Testing:** <Describe what testing was performed and which tests were added.>
Unit test is updated.

**Documentation:** <Describe the documentation added.>
Readme is also updated for awsemfexporter.

### CODE REVIEW

File: exporter/awsemfexporter/util_test.go [hunk 1]
```diff
@@ -182,6 +182,7 @@ func TestGetLogInfo(t *testing.T) {
 			Labels: map[string]string{
 				"aws.ecs.cluster.name": "test-cluster-name",
 				"aws.ecs.task.id":      "test-task-id",
+				"NodeName":             "ip-192-168-58-245.ec2.internal",
```
Comment (mxiamxia) [inline exporter/awsemfexporter/util_test.go:185]: should the we also test `k8s.node.name` as key?

Comment (pxaws) [reply exporter/awsemfexporter/util_test.go]: Sure.

File: exporter/awsemfexporter/util_test.go [hunk 2]
```diff
@@ -173,19 +173,39 @@ func TestGetNamespace(t *testing.T) {
...
+		{
+			Node: &commonpb.Node{
+				ServiceInfo: &commonpb.ServiceInfo{Name: "test-emf"},
+				LibraryInfo: &commonpb.LibraryInfo{ExporterVersion: "SomeVersion"},
+			},
+			Resource: &resourcepb.Resource{
+				Labels: map[string]string{
+					"ClusterName":   "test-cluster-name",
+					"TaskId":        "test-task-id",
+					"k8s.node.name": "ip-192-168-58-245.ec2.internal",
```
Comment (mxiamxia) [inline exporter/awsemfexporter/util_test.go:199]: can we reverse this line with line 186?

Comment (pxaws) [reply exporter/awsemfexporter/util_test.go]: Sure. It will look more symmetric.

**Review by mxiamxia — APPROVED** [review]
Thanks!

**Review by anuraaga — APPROVED** [review]

**Pull request merged**